src/libraries/Native/Unix/System.Native/pal_string.c

```diff
@@ -23,3 +25,12 @@ int32_t SystemNative_SNPrintF(char* string, int32_t size, const char* format, ..
     va_end(arguments);
     return result;
 }
+
+int32_t SystemNative_PrintF(const char* format, ...)
```

Comment:
I think it would be better to expose this as `SystemNative_Log` that just prints string. There is an existing OSX-specific `SystemNative_Log`.